src/demo/index.tsx

```diff
@@ -47,7 +49,7 @@ function SelectApp(props: SelectAppProps): React.ReactElement {
     const {uuid} = props;
     let selectedApp: JSX.Element;
 
-    switch (uuid) {
+    switch (uuid.replace('-nodir', '')) {
```

Comment:
Treating IDs as anything other than totally opaque always makes me nervous! 😛

Suggest do the pre-processing on the UUID in one place - so move this up to the same place as we remove color from the UUID, and see if there's a sensible name instead of `uuid` on `SelectAppProps` since what we're dealing with is no longer really a UUID

Reply:
What about removing the hack altogether and explicitly referencing every uuid? There aren't that many after all:
```tsx
    switch (uuid) {
        case 'fdc3-launcher':
        case 'fdc3-launcher-nodir':
            selectedApp = <LauncherApp />;
            break;
```

Reply:
Your call on that. We're already doing string manipulation to strip out the colour anyway, so could go either way